### ISSUE TITLE
`torch.optim.lr_scheduler.ConstantLR`'s description

### ISSUE THREAD

**Issue by kuraga** [pytorch/pytorch#120716]
### 📚 The doc issue

`torch.optim.lr_scheduler.ConstantLR`

> Decays the learning rate of each parameter group by a small constant factor until the number of epoch reaches a pre-defined milestone: total_iters.

Its example:

> ```python
> # Assuming optimizer uses lr = 0.05 for all groups
> # lr = 0.025   if epoch == 0
> # lr = 0.025   if epoch == 1
> # lr = 0.025   if epoch == 2
> # lr = 0.025   if epoch == 3
> # lr = 0.05    if epoch >= 4
> scheduler = ConstantLR(self.opt, factor=0.5, total_iters=4)
> ```

`Decays ... until epoch reaches total_iters` is supposed to decrease LR on the epoch 1, on the epoch 2, ..., on the epoch total_iters, isn't it?

### Suggest a potential alternative/fix

Maybe, `it sets LR to ... until ...`?

cc @svekars @brycebortree

**Comment by kuraga** [comment]
Also, what do I see on example above?
```python
optimizer = torch.optim.AdamW(model.parameters(), 0.05)
scheduler = torch.optim.lr_scheduler.ConstantLR(optimizer, factor=0.5, total_iters=4)
for epoch in range(7):
    print((epoch, scheduler.get_lr()))
    scheduler.step()
```
```
(0, [0.0125])
(1, [0.025])
(2, [0.025])
(3, [0.025])
(4, [0.1])
(5, [0.05])
(6, [0.05])
```
Don't understand...

**Comment by janeyx99** [comment]
No, the lrs are multiplied by the factor 0.5 for the first 4 iterations, which is what the "Decays" here means. If you think there is a better way to phrase the docs, I'm open to ideas!
